modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -782,9 +782,15 @@ class CV_EXPORTS_W VideoCapture
      */
     CV_WRAP String getBackendName() const;
 
+    /**
+     * raise Error::StsError if a method is not successful
```

Comment:
> `Sts::Error`

Probably we don't need to declare such limitations.

"Switches into exceptions mode - raise exception if some requested operation is failed" ?